src/styles/parts/page.scss

```diff
@@ -0,0 +1,12 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
The universal selector '*' is used here to apply box-sizing to all elements. This is a performance issue. Although it's a common practice to use the universal selector for box-sizing, it's not recommended for large scale projects due to its impact on performance. It would be better to apply box-sizing to specific elements that need it.